weave

```diff
@@ -813,6 +813,16 @@ dns_args() {
     fi
 }
 
+rewrite_hosts() {
+    path=$(docker inspect -f '{{.HostsPath}}' $CONTAINER)
+    name=$(docker inspect -f '{{.Config.Hostname}}' $CONTAINER)
+    containers_path="$(dirname $(dirname "$path"))"
+    docker run --rm \
+        -v "$containers_path":"$containers_path" \
+        --entrypoint=/bin/sh \
+        $EXEC_IMAGE -c "/usr/bin/weavehosts $path $name $ALL_CIDRS"
```

Comment:
> Not sure that using a fixed path is safer than a "random" path..

It is safer because it prevents accidentally mounting over some dir needed by the container.